rtic-monotonics/Cargo.toml

```diff
@@ -63,3 +67,1350 @@ nrf52840 = ["dep:cortex-m", "dep:nrf52840-pac", "dep:critical-section"]
 nrf5340-app = ["dep:cortex-m", "dep:nrf5340-app-pac", "dep:critical-section"]
 nrf5340-net = ["dep:cortex-m", "dep:nrf5340-net-pac", "dep:critical-section"]
 nrf9160 = ["dep:cortex-m", "dep:nrf9160-pac", "dep:critical-section"]
+
+# STM32 timers
+# Use as `features = ["stm32g081kb", "tim15"]`
+tim2 = []
```

Comment:
Please prepend these with `stm32_` to not mix them with other impls in the future. :)

Reply:
`features = ["stm32g081kb", "tim15"]` looked nicer than `features = ["stm32g081kb", "stm32_tim15"]` but sure there might be name clashes in the future, i'll fix it

Reply:
done